pkg/cloudprovider/providers/vsphere/OWNERS

```diff
@@ -0,0 +1,4 @@
+assignees:
```

Comment:
Sorry I got this wrong previously-- actually we're asking maintainers to make this line say "maintainers:"

If you can make this change I'll LGTM :)